src/_common/media-bar/media-bar.ts

```diff
@@ -19,7 +19,7 @@ import AppMediaBarLightbox from './lightbox/lightbox.vue';
 })
 export default class AppMediaBar extends Vue {
 	@Prop(Array) mediaItems!: any[];
-	@Prop(Boolean) noOverlayScrollbars?: boolean;
+	@Prop(Boolean) thinScrollbar?: boolean;
```

Comment:
I think you can get rid of this prop and just always style it as thin. I think we only had this for game jolt sites, which should now work fine with our normal scrollbar styling. We will just have to remember to check a game jolt site when we push.